docs/generator-template.md

```diff
@@ -85,7 +85,7 @@ The **package.json** file is used to define the dependencies for your template.
   "description": "A template that generates a Python MQTT client using MQTT.",
   "generator": {
     "renderer": "react",
-    "apiVersion": "v1",
+    "apiVersion": "v2",
```

Comment:
did you check the code from this tutorial? parser functions that are used are also in v2?

Reply:
Ahh, no 😆 Reverting!